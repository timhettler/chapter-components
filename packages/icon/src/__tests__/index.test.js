import React from 'react';
import { Chevron } from '../';
import renderer from 'react-test-renderer';

it(`Chevron renders correctly`, () => {
  const tree = renderer.create(<Chevron />).toJSON();
  expect(tree).toMatchSnapshot();
});
