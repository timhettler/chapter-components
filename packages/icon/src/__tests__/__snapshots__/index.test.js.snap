// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chevron renders correctly 1`] = `
<svg
  className="icon_style"
  height="100%"
  viewBox="0 0 14 8"
  width="100%"
>
  <polyline
    points="0,1 6,7 7,8 8,7 14,1 13,0 7,6 1,0 "
  />
</svg>
`;
